README.md

```diff
@@ -49,7 +49,39 @@ languages and are part of the [SonarQube](http://www.sonarqube.org/), [SonarClou
 
 ## How to contribute
 
-Check out the [contributing](CONTRIBUTING.md) page to see the best places to log issues and start discussions.
+There are many ways you can contribute to the `sonar-dotnet` project, some very easy and others more involved.
+When contributing, please respect our [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+### Join the discussions
+
+One of the easiest ways to contribute is to participate in discussions on this [GitHub project](https://github.com/SonarSource/sonar-dotnet/issues)

```

Comment:
Is it really a discussion on the `issues` page ?

Reply:
I rephrased it. still not fully satisfied about it